es8/2017-03/mar-21.md

```diff
@@ -0,0 +1,1457 @@
+
+## 3. Adopting the agenda
+
+DT: WeakReferences. Please add 2 additional minutes.
+
+AK: Please add items 1 week in advance.
+
+Agenda adopted
+
+## 4. Adopting the minutes
+
+Minutes approved
+
+## 5. Report from the Ecma Secretariat
+
+AWB: Istvan will do this later
+
+## 6. ECMA262 Status Updates
+
+(Brian Terlson)
+
+BT: Spec is now frozen to any normative changes.
+
+BT: Most new things will go into ES2018
+
+BT: Ready to approve the draft. Minor editorial changes are ok.
+
+BT: Printable Style sheets not working, we need to improve this. Asking browser devs to make printing support easier :)
+
+AWB: the patent document is out for review. Members should get it to their lawyers.
+
+## 7. ECMA402 Status Updates
+
+(Zibi Braniecki)
+
+ZB: I'm not prepared for this discussion. I believe Caridy will be giving this. Will come back in a few minutes
+
+6.ii Vote to accept ECMA-262 2017 final draft and forward it to Ecma GA (approval subject to completion of RF opt-out period with no opt-outs)
+
+AWB: Voting on the standards is one of the few things that needs a vote.  In a minute or two, we will go around and people who actually represent ECMA members should determine who from each organization will vote. Each member will vote yes/no/abstain (where abstain is the equivalent of "no").  We're voting on the 2017 version of ECMA 262 and ECMA 402, based on the final drafts that the editors prepared, subject to the assumption that no one raises an objection during the RF opt-out period (in which case we would not be approving the standard in June, but that has never happened). After we approve it, this will go through the ECMA executive committee, onto the GA in June with the final vote.
+
+WH: No one has ever opted out of an opt-out, but we've had patent shenanigans in the General Assembly before.
+
+MM: Have we had any patent shenanigans regarding ECMAScript?
+
+WH: One of the companies decided to say that they might have patents regarding every standard that was up for a vote.
+
+YK: Only once a feature gets to stage 4 does it get in the spec. What we're doing now is technically a formality.
+
+AWB: I suggest we do a single vote covering both docs.
+
+LBR: We also have the ECMA TR104.
+
+AWB: We will handle that separately.
+
+AWB: Let's go around the room and have those who have voting authority make your vote. Note takers please take not of the organization who is voting.
+
+Tilde: yes
+
+DH: Mozilla Yes
+
+WH: Google yes
+
+Shape security: Yes
+
+FB: Yes
+
+MS: Apple: Yes
+
+GoDaddy: Yes
+
+BT: Microsoft: Yes
+
+JSFoundation: Yes
+
+Intel: Yes
+
+INRIA: Yes
+
+KD: PayPal: Yes
+
+DFV: SalesForce: Yes
+
+AWB: Someone will count them. We got "yesses" and no "no"s.  It will be referred to General Assembly for approval. Our work is done for last year. 
+
+All: _clapping_
```

Comment:
There should be a complete conclusion/resolution section here that includes the resolution to submit the spec for GA approval.

```
#### Conclusion/Resolution

- ....
```